pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -70,15 +70,55 @@ func NewCache() *Cache {
 type NodeCache struct {
 	mu    sync.RWMutex
 	cache predicateMap
+	// Generations for predicates, incremented on predicate invalidation.
+	// Created on first update. Use 0 if does not exist.
+	generations map[string]uint64
```

Comment:
The typical comment style is: "generations is ..."

Reply:
Updated.